src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -520,6 +516,23 @@ private async Task InitializeWorkersAsync()
                 }));
         }
 
+        internal async Task InitializeRpcServiceAsync(IRpcServer rpcService)
+        {
+            _rpcService = rpcService;
+
+            using (_metricsLogger.LatencyEvent(MetricEventNames.HostStartupGrpcServerLatency))
+            {
+                try
+                {
+                    await _rpcService.StartAsync();
+                }
+                catch (Exception grpcInitEx)
+                {
+                    throw new HostInitializationException($"Failed to start Grpc Service. See https://docs.microsoft.com/en-us/azure/app-service/app-service-diagnostics to check if your app is hitting connection limits.", grpcInitEx);
```

Comment:
I could not find a fwd link for this Url.

Reply:
Not sure how useful that link actually is. E.g. if I search the page for "connection limits" I don't find anything. Id just skip the link - your call.